### PR TITLE
Removes font size hardcode from confusion matrix

### LINKED ISSUE #301: ConfusionMatrix FontSize
The confusion matrix label font size is hardcoded to 8pt: https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/classifier/confusion_matrix.py#L228

It appears that this may be a tad too small: https://stackoverflow.com/questions/47450804/yellowbrick-increasing-font-size-on-yellowbrick-generated-charts

We should probably remove this line and make it default to the matplotlib font size.

### PR DESCRIPTION
**Pull request by bbengfort**
Fixes #301

Still required:

- [ ] test improvements (spelling, pytest implementation, more integration tests)
- [ ] tests for confusion matrix attributes 
- [ ] tests for deprecation warnings 
- [ ] update the documentation 

Considering adding a font-size attribute here to make it a bit easier to read.

### CODE REVIEW

File: tests/test_classifier/test_confusion_matrix.py [hunk 1]
```diff
@@ -1,138 +1,279 @@
...
+        ylabels = [l.get_text() for l in ax.get_yticklabels()]
+        ylabels.reverse()
+        assert  ylabels == classes
 
     def test_isclassifier(self):
         """
-        Test taht non-classifiers raise exceptions
+        Assert that only classifiers can be used with the visualizer.
         """
         model = PassiveAggressiveRegressor()
```
Comment (ndanielsen) [inline tests/test_classifier/test_confusion_matrix.py]: golden!

File: yellowbrick/__init__.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 _orig_rc_params = mpl.rcParams.copy()
 
 # Import the version number at the top level
-from .version import get_version
+from .version import get_version, __version_info__
```
Comment (ndanielsen) [inline yellowbrick/__init__.py]: great, this is very needed

File: tests/test_classifier/test_confusion_matrix.py [hunk 3]
```diff
@@ -1,138 +1,279 @@
...
-        classes = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
+        message = (
+            'This estimator is not a classifier; '
+            'try a regression or clustering score visualizer instead!'
+        )
 
-        with self.assertRaisesRegexp(yellowbrick.exceptions.YellowbrickError, message):
-            ConfusionMatrix(model, classes=classes)
+        with self.assertRaisesRegexp(yb.exceptions.YellowbrickError, message):
+            ConfusionMatrix(model)
```
Comment (ndanielsen) [inline tests/test_classifier/test_confusion_matrix.py]: the one ask is to add pandas support and a test for it

Comment (bbengfort) [reply tests/test_classifier/test_confusion_matrix.py]: Pandas test added!

**Pull request merged**